backend/src/main/kotlin/vegait/rs/osipodgorica/service/CategoryService.kt

```diff
@@ -19,11 +25,42 @@ class CategoryService(val categoryRepo: CategoryRepository, val uploadService: I
         return categoryRepo.save(category)
     }
 
+    fun update(request: UpdateCategoryRequest, id: Long) =
+        runCatching {
+            categoryRepo.findById(id).get()
```

Comment:
Možemo ovo da pojednostavimo, tako što bacamo exception kada ne pronađemo element u bazi 

var category = categoryRepo.findById(id).orElseThrow()

U ovom slučaju ćemo dobiti kategoriju ako bude pronađena, u suprotnom se baca exception koji će spring ishendlati za sad, kasnije dodajemo pravu konf za exception handling

Reply:
Pa `get()` radi isto, isti Exception varaca, a `.getOrThrow()` na kraju svakako propagira exception. Da nisam nesto pogresno shvatio?